tensorflow/compiler/mlir/lite/ir/tfl_ops.td

```diff
@@ -5364,16 +5364,15 @@ Read variable data identified by 'resource_id'.
 def TFL_Conv3DOp : TFL_Op<"conv_3d", [
     Pure,
     AccumulatorUniformScale<2, 0, 1>,
+    QuantizableResult,
+    AffineQuantizedOpInterface,
+    AffineOpCoefficient<4, 1>,
```

Comment:
Apparently, there are more discussions for which the per channel dimension location should be supported in this operator in the long term.

Could you provide the rationale for your choice here for the future decision?

Reply:
Sorry, I'm not sure to understand. We use per channel quantization for the Conv3D, so we use the output channel dimension (dimension 4) of the filters (operand 1). Though I might have misunderstood the `AffineOpCoefficient` usage.

Reply:
@abattery Can you please respond/resolve this thread? Thanks